obs/api/app/controllers/api/storage.py

```diff
@@ -107,9 +107,13 @@ def post(self, bucket_name):
         try:
             regex = r"[^a-z0-9.-]"
             bucket_name = re.sub(regex, "", bucket_name)
-            if len(bucket_name) < 3:
-                return response(400, f"'{bucket_name}' too short for bucket name")
+            current_app.logger.error(len(bucket_name))
+            if not 2 < len(bucket_name) < 64:
```

Comment:
pass check bucket name between 3 and 63 char. 
* Bucket names must be at least 3 and no more than 63 characters long.